jenkins_jobs/build_host_os/script.sh

```diff
@@ -32,5 +35,9 @@ python host_os.py \
 # creating the yum repository locally
 createrepo ./repository
 
+# Create META file with timestamp information
+touch ./META
```

Comment:
There's no need to "touch" the file, it will be created in the next command anyway. The SUCCESS file you see below is always empty.